java/src/jmri/configurexml/ConfigXmlManager.java

```diff
@@ -121,23 +121,22 @@ public Object findInstance(Class<?> c, int index) {
 
     @Override
     public ArrayList<Object> getInstanceList(Class<?> c) {
```

Comment:
You want to return List instead of ArrayList, since this would allow the consuming method to not be concerned about the type of List it gets, (ArrayList is one of many implementations of List).

Reply:
Not my code… but I will change it. (Uh, git blame says the author was… Randall Wood! ;-)